golem/actions.py

```diff
@@ -513,19 +517,19 @@ def wait_for_element_visible(element, timeout=20):
             timed_out = True
 
 
-def http_get(url, headers={}, params={}):
+def http_get(url, verify_ssl_cert, headers={}, params={}):
```

Comment:
The param verify_ssl_cert should be in the last position and it should be a named parameter with default True which is the default for Requests package (http://docs.python-requests.org/en/master/user/advanced/#ssl-cert-verification)